src/ray/raylet/worker_pool.h

```diff
@@ -144,6 +153,9 @@ class WorkerPool {
   struct State {
     /// The commands and arguments used to start the worker process
     std::vector<std::string> worker_command;
+    /// The pool of lazy workers for actor creation tasks
+    /// with prefix or suffix worker command.
+    std::unordered_map<TaskID, std::shared_ptr<Worker>> idle_lazy_workers;
```

Comment:
How about `on-demand` or `dedicated` workers?

Reply:
maybe `dedicated` is better as this kind of worker is specifically created for a particular task